libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -522,6 +538,23 @@ string YulUtilFunctions::overflowCheckedIntMulFunction(IntegerType const& _type)
 	});
 }
 
+string YulUtilFunctions::wrappingIntMulFunction(IntegerType const& _type)
+{
+	string functionName = "wrapping_mul_" + _type.identifier();
+	return m_functionCollector.createFunction(functionName, [&]() {
+		return
+			// Multiplication by zero could be treated separately and directly return zero.
```

Comment:
What do you mean by that? Check if `x` or `y` is zero at runtime?

Reply:
This was copied from the checked version - it is not relevant here. I will remove it.